tests/unit/lms/services/lti_grading/_v11_test.py

```diff
@@ -74,23 +74,34 @@ def test_record_result(self, svc, score, http_service, score_payload):
     def test_sync_grade(self, svc, http_service, application_instance, db_session):
         lms_user = factories.LMSUser()
         assignment = factories.Assignment()
-        lti_role = factories.LTIRole()
-        db_session.flush()
-        membership = factories.LMSUserAssignmentMembership(
-            lms_user_id=lms_user.id,
-            assignment_id=assignment.id,
-            lti_role_id=lti_role.id,
+        factories.LMSUserAssignmentMembership(
```

Comment:
Adjusting these for a more complete tests. This test now fails without the code changes